### PR TITLE
 added styling

### PR DESCRIPTION
**Pull request by Richez003**
added styling as recommended 
fully functional

### CODE REVIEW

File: src/components/header.css [hunk 1]
```diff
@@ -1,20 +1,65 @@
-* {
-  box-sizing: border-box;
-  padding: 0;
-  margin: 0;
-  color: black;
-}
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Roboto+Slab:wght@100&display=swap');
 
-.head {
+nav {
```
Comment (BenMukebo) [inline src/components/header.css:3]: - [x]  According to the project requirement, Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please make these in order to stick to the original design:
- Kindly make sure the user icon is positional to the right of your navigation bar 😜 
- What if we increase the footer input width in order the form take the whole witdh 😜 

- Your Design:

25last



- Figma design

bookstore

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Richez003,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README ✔️ 
- You have followed  JavaScript best practices. ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x]  Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯 
- [ ] Please the image link is broken, Kindly  fix this
- [ ] Please update your Readme, currently, some headings are missing such as 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse

##  License 
```

``` ##  License ``` section with the MIT license at the end of your README file to make it more professional. I would also recommend you add this. 😄  please refer to this [template](https://github.com/microverseinc/readme-template).

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Please it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser. You can put the steps to clone and run the project in a new Getting Started section under the project's description.


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/header.css (hunk 1 above)
Comment (Richez003) [reply src/components/header.css:3]: done

**Review by mateo951 — APPROVED** [review]
### Hi @Richez003, 

Well done making the changes requested by the previous reviewer 😁
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**